tests/runtests.py

```diff
@@ -241,6 +233,12 @@ def setUp(self):
             if re.match(regex, data.get('@id', data.get('id', ''))):
                 self.skipTest('Test with id regex %s' % regex)
 
+        # mark tests as pending, meaning that they are expected to fail
```

Comment:
Better than skipping tests that aren't ready, as it will fail when it starts passing.